src/content/docs/tutorial-app-performance/reduce-javascript-errors.mdx

```diff
@@ -13,56 +13,53 @@ import browserJsErrorCharts from 'images/browser_screenshot-crop_error-overview.
 
 import browserSingleErrorOverview from 'images/browser_screenshot-crop_error-eventlog-stacktrace-overview.webp'
 
-JavaScript errors can have significant negative impacts on how customers use your site and perceive your brand. From causing user frustration to damaging brand perception, you need to find and fix JavaScript errors as soon as they occur to help maintain site performance and keep your customers happy. New Relic makes pin-pointing these errors straightforward no matter the number or the size, helping you get your site up and running faster and more efficiently.
+JavaScript errors affect how customers use your site and their perception of your brand. You should prioritize fixing JavaScript errors as soon as they occur to help maintain site performance and keep your customers happy. 
 
-## Finding JavaScript errors [#find-errors] 
+## Objectives [#objectives]
+
+This tutorial walks you through finding and resolve JavaScript errors. By the end of the tutorial, you will have these tasks completed:
+
+* Finding JavaScript errors information in New Relic
```

Comment:
Should be periods here for consistency with the objective bullet lists on other pages